scripts/ci/constraints/ci_generate_constraints.sh

```diff
@@ -28,6 +28,4 @@ shift
 
 build_images::prepare_ci_build
 
-build_images::rebuild_ci_image_if_needed_with_group
```

Comment:
This is not needed (it just makes constraint generation longer - we already have the images pulled at this stage)